Code-the-Dream-School/cc-prac-team4-front
# Pp 124 add functionality to forgot password page

**Functionality was added to forgot password page**

**Pull Request Description:**

This pull request adds functionality to the forgot password page. The changes include the following modifications:

1. Added a new action function called forgotPassword in userAction.js that handles the forgot password functionality.
2. Updated the forgotPasswordSubmit function in ForgotPassword.js to use the forgotPassword action.
3. Created a new reducer called forgotPasswordReducer in userReducer.js to handle the state for the forgot password functionality.

These changes allow users to reset their passwords when they forget them.

**Testing Steps:**

**To test the changes made in this pull request, follow these steps:**

- Set up and run the project locally, if not already done.
- Go to the forgot password page. [http://localhost:3000/password/forgot](http://localhost:3000/password/forgot)
- Enter a valid email address and submit the form.
- Check the console logs for any error messages or responses related to the forgot password functionality.
- Verify that the appropriate success or error messages are displayed on the page.

<img width="1175" alt="Screenshot 2023-07-02 at 10 03 34 PM" src="https://github.com/Code-the-Dream-School/cc-prac-team4-front/assets/2952900/dbd6bb16-0cea-4a68-9a3d-a424f39d6341">
